### ISSUE TITLE
Doesn't work anymore

### ISSUE THREAD

**Issue by Vanilagy** [spite/THREE.EquirectangularToCubemap#3]
This doesn't work anymore in new versions of THREE. Valve, please fix.

**Comment by kylebakerio** [comment]
ping

**Comment by Vanilagy** [comment]
pong

**Comment by kylebakerio** [comment]
trying to look into other options. found this, seems like it's probably the key to doing it with modern THREE: https://threejs.org/docs/#api/en/renderers/WebGLCubeRenderTarget.fromEquirectangularTexture

**Comment by kylebakerio** [comment]
yeah, used that here: https://github.com/kylebakerio/aframe-interior-parallax-mapping/blob/master/interior-parallax.js

gross quick hack, thrown together in a couple hours. but it works. need to remove that setTimeout, but not sure what the correct event is to listen for there.
